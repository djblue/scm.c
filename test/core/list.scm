(write (list 1 (+ 1 1) 3))
(write (range 5))
(write (reduce + (range 5) 0))
(write (member 5 (range 5)))
(write (member 0 (range 5)))
(write (map number? (range 2)))
(write (reverse (range 5)))
(write (length (range 5)))
