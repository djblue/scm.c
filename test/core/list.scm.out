(1 2 3)
(0 1 2 3 4)
10
#f
(0 1 2 3 4)
(#t #t)
(4 3 2 1 0)
5
